client/my-sites/plugins/plugin-meta/index.jsx

```diff
@@ -453,10 +451,11 @@ const PluginMeta = React.createClass( {
 							</div>
 							{ ! isEmpty( actionLinks ) &&
 								<div className="plugin-meta__action-links">
-									{ Object.keys( actionLinks ).map( linkTitle => (
+									{ Object.keys( actionLinks ).map( ( linkTitle, index ) => (
 										<Button compact icon
 											href={ actionLinks[ linkTitle ] }
 											target="_blank"
+											key={ 'action-link-' + index }
```

Comment:
How about using the `linkTitle` as the key?
https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318